langs/langs.json

```diff
@@ -3,6 +3,7 @@
   { "code": "az", "name": "Azərbaycanca", "enName": "Azerbaijani" },
   { "code": "be", "name": "Беларуская", "enName": "Belarusian" },
   { "code": "bn", "name": "বাংলা", "enName": "Bengali" },
+  { "code": "ckb", "name": "کوردی ناوڕاست", "enName": "Central Kurdish" },
```

Comment:
```suggestion
  { "code": "ckb", "name": "کوردیی ناوەندی", "enName": "Central Kurdish" },
```

Reply:
thanks for your care, 
resolved :heavy_check_mark: